src/module/item/consumable/data.ts

```diff
@@ -43,7 +43,17 @@ interface ConsumableSystemData
         Omit<PhysicalSystemData, "traits"> {
     stackGroup: AmmoStackGroup | null;
 }
-type SourceOmission = "bulk" | "description" | "hp" | "identification" | "material" | "price" | "temporary" | "usage";
+
```

Comment:
type SourceOmission = "bulk" | "description" | "hp" | "identification" | "material" | "price" | "temporary" | "usage";

Can save a few lines by just putting `apex?: never` in both system source and system data